app/views/partials/service_relationships.scala.html

```diff
@@ -81,3 +82,10 @@
     }
     </ul>
 }
+
+@popoverContent(lifecycleStatus: Option[LifecycleStatus]) = {
+    @lifecycleStatus match {
+      case Some(Deprecated) => { "Marked as deprecated" }
```

Comment:
Should we just say "Deprecated"? The "Marked" terminology was introduced for decommissioning to show that it's still in the process and not decommissioned just yet.